src/main/java/net/mcreator/plugin/PluginLoader.java

```diff
@@ -88,6 +90,21 @@ public PluginLoader() {
 				}
 
 				plugin.loaded = true;
+
+				if(MCreatorApplication.isInternet) {
+					if (plugin.getInfo().getUpdateJsonUrl() != null) {

```

Comment:
All this parsing should have own try in case plugin update URL is down or returns invalid JSON so this does not break MCreator plugin loading.